mevislab.github.io/content/tutorials/visualization/visualizationexample8.md

```diff
@@ -0,0 +1,147 @@
+
+By the end of this process, you'll have two images at your dispose. 
+
+![Enhanced Skeletonization Results (With Colors)](/images/tutorials/visualization/V8_08.png "Enhanced Skeletonization Results (With Colors)")
+
+### Interaction with the Generated Vascular System Using SoVascularSystem
+
+If you wish to engage with the generated vascular system interactively, the `SoVascularSystem` module is your gateway to seamless exploration and interaction. Connect it to your `DtfSkeletonization` module, explore its settings and observe the changes..
+
+![ SoVascularSystem](/images/tutorials/visualization/V8_09.png " SoVascularSystem")
+
+### Enhance Vessel Visualization Based on Distance Information
+
+Now that you've successfully obtained the vessel skeleton graph using `DtfSkeletonization`, let's take the next step to enhance the vessel visualization based on the distance information. We'll modify the existing code to incorporate the Skeleton's property **Label** for storing distance information, which will subsequently be used to color the rendering.
+
+In your `RunPythonScript` module, replace the existing code with the following:
+
+{{< highlight >}}
+```Python
+
+ctx.field("DtfSkeletonization.update").touch()
+
+graph = ctx.field("DtfSkeletonization.outBase1").object()
+if graph is not None:
+  label = "Label"
+  print('Num edges', len(graph.getEdges()))
+  for edge in graph.getEdges():
+    end_node = edge.getEndNode()
+    for skeleton in edge.getSkeletons():
+      if label not in skeleton.properties:
+        skeleton.createPropertyDouble(label, edge.getId())
+      skeleton.setProperty(label, skeleton.getProperty("MinDistance"))
+
+  ctx.field("GraphToVolume.update").touch()
+  ctx.field("SoVascularSystem.apply").touch()
+
+  {{</highlight>}}
+
+
+{{<alert class="info" caption="Extra Infos">}}
+
+This modified script ensures that the Skeleton's property **Label** is utilized to store its distance information. The vessels will now be displayed with colors based on their minimal distance. 
+
+{{</alert>}}
+
+Now, let's visualize the impact of the modifications in the script. After executing the updated network, head to the `SoLUTEditor` module in the interface. Once there, navigate to the *Range* tab and tweak the *New Range Max* to *10*.
+
+With these adjustments made, click on *Apply New Range*. Immediately, you'll observe a dynamic transformation in the color representation of the vessel visualization. This alteration, driven by the minimal distance, enhances the clarity and informativeness of the displayed vascular structures. Take this opportunity to explore and analyze the results, providing valuable insights into the intricacies of the vessel system.
+
+![ Enhanced_Vessel_Visualization_Result](/images/tutorials/visualization/V8_010.png "Enhanced_Vessel_Visualization_Result")
```

Comment:
show some visualization examples of the SoVascularSystem module, the difference to SoGVRVolumeRenderer is, that we can visualize special vascular things like centerline only and points instead of just rendering a mask in 3D like SoGVRVolumeRenderer